pvlib/pvsystem.py

```diff
@@ -1887,6 +1887,82 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
+def pvsyst_celltemp(
+    poa_global,
+    wind_speed,
+    temp_air,
+    eta_m=0.1,
+    alpha_absorption=0.9,
+    temp_model="freestanding",
+):
+    """
+    Calculate PVSyst cell temperature.
+
+    Parameters
+    ------------
+    poa_global : float or Series
```

Comment:
I believe that all of these `float or Series` parameters can be better described as `numeric` (they can also be arrays or even DataFrames).

Reply:
I've got these formatting changes ready for the next commit. When should I use the "resolve conversation" button? After I commit, before committing as I check off each item or let you do it once you're satisfied?